### PR TITLE
Implement postgraphile subscriptions for the updated schema

### PR DESCRIPTION
**Pull request by arijitAD**
fixes:https://github.com/vulcanize/ipld-eth-db/issues/20

### CODE REVIEW

**Review by i-norden — COMMENTED** [review]
LGTM just one change requested and a question, thanks!

**Comment by i-norden** [comment]
Thanks @arijitAD ! Something weird going on with goose in the CI build. I think that's just an artifact?

File: db/migrations/00016_create_postgraphile_triggers.sql [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+END;
+$$ language plpgsql;
+-- +goose StatementEnd
+
+CREATE TRIGGER trg_eth_header_cids
+    AFTER INSERT ON eth.header_cids
+    FOR EACH ROW
+    EXECUTE PROCEDURE eth.graphql_subscription();
+
+CREATE TRIGGER trg_uncle_cids
```
Comment (i-norden) [inline db/migrations/00016_create_postgraphile_triggers.sql:65]: Looks like this needs to be `trg_eth_uncle_cids` https://github.com/vulcanize/ipld-eth-db/runs/4580319689?check_suite_focus=true#step:5:104

File: db/post_batch_processing_migrations/00020_create_postgraphile_triggers.sql [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+END;
+$$ language plpgsql;
+-- +goose StatementEnd
+
+CREATE TRIGGER trg_eth_header_cids
+    AFTER INSERT ON eth.header_cids
+    FOR EACH ROW
+    EXECUTE PROCEDURE eth.graphql_subscription();
+
+CREATE TRIGGER trg_uncle_cids
```
Comment (i-norden) [inline db/post_batch_processing_migrations/00020_create_postgraphile_triggers.sql:65]: Here too

**Review by i-norden — COMMENTED** [review]
One of the trigger names is throwing an error, looks good otherwise.

File: db/migrations/00016_create_postgraphile_triggers.sql (hunk 1 above)
Comment (arijitAD) [reply db/migrations/00016_create_postgraphile_triggers.sql]: Done.

File: db/post_batch_processing_migrations/00020_create_postgraphile_triggers.sql (hunk 2 above)
Comment (arijitAD) [reply db/post_batch_processing_migrations/00020_create_postgraphile_triggers.sql]: Done.

**Pull request merged**